blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -126,6 +126,10 @@ Changes
 Fixes
 =====
 
+- Fail the storage engine if indexing on a replica shard fails after it was
```

Comment:
well `storage engine` doesn't seem to be user facing, but the second sentence explains a bit better why it is done.

Reply:
@seut or @mfussenegger can you please take a look at this PR and especially the changelog entry?

Reply:
Agree that the storage engine detail might not be useful to users, but since it also contains a higher level issue description I think it is fine.